server/api.js

```diff
@@ -6,6 +6,7 @@ api
   .get('/heartbeat', (req, res) => res.send({ok: true}))
   .use('/auth', require('./auth'))
   .use('/users', require('./users'))
+  .use('/reviews', require('./review'))
```

Comment:
please change the file name to reviews to stay in line with previous things like users.